rotkehlchen/tests/api/test_metadata.py

```diff
@@ -21,7 +21,9 @@ def test_metadata_endpoint(rotkehlchen_api_server: 'APIServer') -> None:
             'name': airdrop.name,
             'icon': airdrop.icon,
         }
-        for identifier, airdrop in AIRDROPS.items()
+        for identifier, airdrop in fetch_airdrops_metadata(
```

Comment:
shouldn't this be VCRed? It always does a githu query? Or at least somehow mocked?